resources/containers/styles/button.module.css

```diff
@@ -0,0 +1,36 @@
+.button {
```

Comment:
Dont put a styles folder, it can be at the same level.

The fact that both buttons have the same style means they could be the same component, just handling the input as a prop. 

1 Style - 1 Component rule